core/controllers/improvements_test.py

```diff
@@ -637,7 +637,12 @@ def test_get_returns_exploration_version(self):
         self.assertEqual(json_response['exploration_version'], 1)
 
         # Update to version 2.
-        exp_services.update_exploration(self.owner_id, self.EXP_ID, None, '')
+        exp_services.update_exploration(
+            self.owner_id,
+            self.EXP_ID,
+            None,
+            ''
```

Comment:
Can we please enforce these two args to be named? Can be done in a subsequent PR.